setup.py

```diff
@@ -24,10 +24,12 @@
 except:
     requirements.append("hashlib")
 
+import bagit
```

Comment:
ahh ok. I don't do as much python as other languages so I probably am miss-remembering. Doesn't matter too much to me. I would say with your first merge that it looks good to me.